sqsjobs/item.go

```diff
@@ -148,10 +157,7 @@ func (i *Item) Ack() error {
 	return nil
 }
 
-func (i *Item) Nack() error {
-	if atomic.LoadUint64(i.Options.stopped) == 1 {
-		return errors.Str("failed to acknowledge the JOB, the pipeline is probably stopped")
-	}
+func commonNack(i *Item) error {
```

Comment:
commonNack should be on the Item structure (no need to pass i):
```go
func (i *Item) commonNack() error {}
```

Reply:
I can't get this to work. If I change the signature it says

```go
sqsjobs/item.go:211:9: undefined: commonNack
sqsjobs/item.go:227:9: undefined: commonNack (typecheck)
```

Reply:
Yeah, because it should be called like: `i.commonNack`